public/components/datasources/components/manage/data_connection.tsx

```diff
@@ -172,18 +172,64 @@ export const DataConnection = (props: any) => {
       disabled: false,
       content: (
         <AssociatedObjectsTab
-          associatedObjects={
-            [
-              // {
-              //   id: '1',
-              //   name: 'Table_name_1',
-              //   database: 'db1',
-              //   type: 'Table',
-              //   createdByIntegration: 'xx',
-              //   accelerations: 'xxx_skipping',
-              // },
-            ]
-          }
+          associatedObjects={[
```

Comment:
could you move these test data (if required) to a somewhat mock file and exporting it?

Reply:
done.